R/type-list-of.R

```diff
@@ -30,61 +119,100 @@ as_list_of <- function(x, ...) {
 }
 
 #' @export
-as_list_of.vctrs_list_of <- function(x, ..., .ptype = NULL) {
-  if (!is.null(.ptype)) {
-    x <- unclass(x)
-    list_as_list_of(x, ptype = .ptype)
-  } else {
-    x
-  }
+as_list_of.vctrs_list_of <- function(x, ...) {
+  x
 }
```

Comment:
Breaking change, removal of `.ptype` for this method. It will be a silent no-op if this arg was specified